maps/map_files/LV624/LV624.dmm

```diff
@@ -18911,17 +18936,6 @@
 	icon_state = "grassdirt_corner"
 	},
 /area/lv624/ground/colony/west_nexus_road)
-"nOw" = (
```

Comment:
i saw it was weird but i was thinking it was a legacy item i could delete. tell me if i am wrong.